test-cases/longevity/longevity-50GB-4days-authorization-and-tls-ssl.yaml

```diff
@@ -5,11 +5,13 @@ stress_cmd: ["cassandra-stress mixed cl=QUORUM duration=5760m -schema 'replicati
              "cassandra-stress counter_write cl=QUORUM duration=5760m -schema 'replication(factor=3) compaction(strategy=DateTieredCompactionStrategy)' -port jmx=6868 -mode cql3 native -rate threads=1 -pop seq=1..1000000",
              "cassandra-stress user profile=/tmp/cs_mv_profile.yaml ops'(insert=3,read1=1,read2=1,read3=1)' cl=QUORUM duration=5760m -port jmx=6868 -mode cql3 native -rate threads=10",
              "cassandra-stress write cl=QUORUM duration=5760m -schema 'replication(factor=3) compression=LZ4Compressor compaction(strategy=SizeTieredCompactionStrategy)' -port jmx=6868 -mode cql3 native compression=lz4 -rate threads=50 -pop seq=1..100000000 -log interval=5",
+             "cassandra-stress write cl=QUORUM duration=5760m -schema 'replication(factor=3) compression=ZstdCompressor compaction(strategy=SizeTieredCompactionStrategy)' -port jmx=6868 -mode cql3 native compression=none -rate threads=50 -pop seq=1..100000000 -log interval=5",
              "cassandra-stress write cl=QUORUM duration=5760m -schema 'replication(factor=3) compression=SnappyCompressor compaction(strategy=SizeTieredCompactionStrategy)' -port jmx=6868 -mode cql3 native compression=snappy -rate threads=50 -pop seq=1..100000000 -log interval=5",
              "cassandra-stress write cl=QUORUM duration=5760m -schema 'replication(factor=3) compression=DeflateCompressor compaction(strategy=SizeTieredCompactionStrategy)' -port jmx=6868 -mode cql3 native compression=none -rate threads=50 -pop seq=1..100000000 -log interval=5"]
 stress_read_cmd: ["cassandra-stress read cl=QUORUM duration=5760m -port jmx=6868 -mode cql3 native  -rate threads=50 -pop seq=1..100000000 -log interval=5",
                   "cassandra-stress counter_read cl=QUORUM duration=5760m -port jmx=6868 -mode cql3 native -rate threads=10 -pop seq=1..1000000",
                   "cassandra-stress read cl=QUORUM duration=5760m -schema 'replication(factor=3) compression=LZ4Compressor compaction(strategy=SizeTieredCompactionStrategy)' -port jmx=6868 -mode cql3 native compression=lz4 -rate threads=20 -pop seq=1..100000000 -log interval=5",
+                  "cassandra-stress read cl=QUORUM duration=5760m -schema 'replication(factor=3) compression=ZstdCompressor compaction(strategy=SizeTieredCompactionStrategy)' -port jmx=6868 -mode cql3 native compression=none -rate threads=20 -pop seq=1..100000000 -log interval=5",
```

Comment:
the amount of loader we currently have in this test, is enough for adding this extra stress command ?

Reply:
I have run it couple of times, and it didn't feel too much.
It could have implications if instance type changes over time or if massive workload on same network could impact load.

Reply:
@fruch

Reply:
Should be ok, if we face an issue we can use round_robin.